src/Nelmio/Alice/Fixtures.php

```diff
@@ -249,7 +249,7 @@ private function validateOptions(array $options)
                 if (false === is_object($provider) && false === is_string($provider)) {
                     throw new \InvalidArgumentException(sprintf(
                         'The provider should be a string or an object, got %s instead',
-                        is_scalar($provider) ?$provider :gettype($provider)
+                        is_scalar($provider) ? $provider : gettype($provider)
```

Comment:
Personally, I prefer it as well. We do have a CS checker, but I'm not sure if it handles this situation.